services/repository/adopt.go

```diff
@@ -146,6 +146,12 @@ func adoptRepository(ctx context.Context, repoPath string, repo *repo_model.Repo
 	}
 	defer gitRepo.Close()
 
+	objectFormat, err := gitRepo.GetObjectFormat()
+	if err != nil {
+		return fmt.Errorf("getObjectFormat: %w", err)
+	}
+	repo.ObjectFormatName = objectFormat.Name()
+
```

Comment:
`SyncRepoBranchesWithRepo` already syncs object format.

Reply:
f887ed08074cc7b7723b8ba0a85c69835cc1bddd
I think the column has been updated twice when adopted. The first time it will be updated to the correct content, but continued steps will update it to blank again.